forestflow/model_p3d_arinyo.py

```diff
@@ -246,6 +246,23 @@ def P3D_Mpc(self, z, k, mu, pp):
         D_NL = np.exp(nonlin * (1 - vel) - press)
 
         return linP * lowk_bias**2 * D_NL
+    
+    
+    def Px_Mpc(self, z, k_par, pp):
```

Comment:
I think it would be interesting to have rperp as an input parameter to this function, is that possible?